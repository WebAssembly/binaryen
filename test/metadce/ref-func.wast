(module
 (import "env" "f1" (func $f1))
 (import "env" "f2" (func $f2))

 (export "g" (global $g))
 (export "f" (func $f))

 (global $g funcref (ref.func $f1))

 (func $f (result funcref) (ref.func $f2))
)
