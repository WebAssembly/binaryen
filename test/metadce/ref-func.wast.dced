(module
 (type $none_=>_none (func))
 (type $none_=>_funcref (func (result funcref)))
 (import "env" "f2" (func $f2 (type $none_=>_none)))
 (import "env" "f1" (func $f1 (type $none_=>_none)))
 (global $g funcref (ref.func $f1))
 (elem declare func $f2)
 (export "g" (global $g))
 (export "f" (func $f))
 (func $f (type $none_=>_funcref) (result funcref)
  (ref.func $f2)
 )
)
