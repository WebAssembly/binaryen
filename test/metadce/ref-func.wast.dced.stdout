unused: f1
unused: f2
